GRIB2_Template_4_126_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,52 @@
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",67,n - number of time range specifications describing the time intervals used to calculate the statistically processed field,,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",68-71,Total number of data values missing in statistical process,,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",,72-83   Specification of the outermost (or only) time range over which statistical processing is done,,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",72,Statistical process used to calculate the processed field from the field at each time increment during the time range,(see Code table 4.10),,4.1,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",73,Type of time increment between successive fields used in the statistical processing,(see Code table 4.11),,4.11,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",74,Indicator of unit of time for time range over which statistical processing is done,(see Code table 4.4),,4.4,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",75-78,"Length of the time range over which statistical processing is done, in units defined by the previous octet",,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",79,Indicator of unit of time for the increment between the successive fields used,(see Code table 4.4),,4.4,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",80-83,"Time increment between successive fields, in units defined by the previous octet",(see Notes 3 and 4),"7,99",,,Operational
```

Comment:
7,"Above normal, near normal and below normal are defined as three equiprobable categories based on climatology at each point over the geographical area covered by the grid. The type and methodology of the reference climatology are unspecified and should be documented concurrently by the data producer."